tests/test_climate.py

```diff
@@ -316,6 +316,22 @@ async def test_climate_local_temperature(
     assert entity.state["current_temperature"] == 21.0
 
 
+async def test_climate_outdoor_temperature(
+    device_climate: Device,
+    zha_gateway: Gateway,
+) -> None:
+    """Test outdoor temperature."""
+
+    thrm_cluster = device_climate.device.endpoints[1].thermostat
+    entity: ThermostatEntity = get_entity(
+        device_climate, platform=Platform.CLIMATE, entity_type=ThermostatEntity
+    )
+    assert entity.state["outdoor_temperature"] is None
+
+    await send_attributes_report(zha_gateway, thrm_cluster, {1: 2150})
```

Comment:
Thermostat.AttributeDefs.outdoor_temperature.id instead of 1 please. We have a mix now but we are slowly trying to clean them up.